src/app/loadout/loadout-edit/LoadoutEdit.tsx

```diff
@@ -62,16 +70,10 @@ export default function LoadoutEdit({
   loadout,
   store,
   setLoadout,
-  onClickSubclass,
-  onClickPlaceholder,
-  onClickWarnItem,
 }: {
   loadout: Loadout;
   store: DimStore;
   setLoadout: (updater: LoadoutUpdateFunction) => void;
-  onClickSubclass: (subclass: DimItem | undefined) => void;
-  onClickPlaceholder: (params: { bucket: InventoryBucket; equip: boolean }) => void;
-  onClickWarnItem: (resolvedItem: ResolvedLoadoutItem) => void;
```

Comment:
I moved these into LoadoutEdit because I don't really see why there would be different implementations of them at this point, and it helped to move the `useItemPicker` into the Sheet.